tooling/templatize/internal/end2end/e2e_test.go

```diff
@@ -410,3 +410,34 @@ param parameterA = 'Hello Bicep'`,
 	require.NoError(t, err)
 	require.Equal(t, string(io), "Hello Bicep\n")
 }
+
+func TestE2EOutputOnlyFailsIfContainsResources(t *testing.T) {
+	t.Parallel()
+	if !shouldRunE2E() {
+		t.Skip("Skipping end-to-end tests")
+	}
+
+	tmpDir := t.TempDir()
+
+	e2eImpl, err := newE2E(tmpDir, "../../testdata/e2eOutputOnly.yaml")
+	require.NoError(t, err)
+
+	bicepFile := `
```

Comment:
You mentioned a nested module being the trigger, but I don't see that here - did you want this e2e to cover that case?

Reply:
There's no way to run these locally for us, yeah?